inc/field.class.php

```diff
@@ -92,6 +92,9 @@ static function install(Migration $migration, $version) {
       $toolbox = new PluginFieldsToolbox();
       $toolbox->fixFieldsNames($migration, ['NOT' => ['type' => 'dropdown']]);
 
+      //regenerate container files
+      plugin_fields_checkFiles(true);
+
```

Comment:
The is already a call to this function in `plugin_fields_install()`. Maybe the `$force = true` parameter should be added there.
```suggestion
```

Reply:
Yes it's needed to force to update generated file